AndroidSDKFcm/src/main/java/com/leanplum/LeanplumPushFirebaseMessagingService.java

```diff
@@ -22,15 +22,21 @@
 package com.leanplum;
 
 import android.annotation.SuppressLint;
```

Comment:
As per the doc, for iOS there was existing duplicate notification logic. Were you able to investigate the background why it was implemented. The file was LPActionManager.m